src/components/draw/style/draw.less

```diff
@@ -1,111 +1,294 @@
+@ga-select-box-padding: 1;
+.ga-item-size(@width, @height, @nbCol, @nbItems, @btWidth) {
+   /* (@width + 2*margin)*@nbCol + @btWidth + 2*padding */
+   width: unit((@width + 4)*@nbCol + @btWidth + 2*@ga-select-box-padding , px);
+   
+   /* @height + 2*margin + 2*padding + 2*borderWidth */
+   height: unit(@height + 4 + 2*@ga-select-box-padding + 2, px); 
+      
+   &.ga-select-box-open {
+     height: unit(ceil(@nbItems/@nbCol)*(@height + 4) + 2*@ga-select-box-padding, px);
+   }
+   > * {
+     margin: 2px;
+     width: unit(@width, px);
+     height: unit(@height, px);
+   }
+   button {
+     padding-top: unit(@ga-select-box-padding, px);
+   }
+}
+
 [ga-draw] {
+  /* 100% doesn't work so we specify a fix width for the position
+  of the more  button */
+  width: 285px;
+  @media (max-width: @screen-tablet), (max-height: @screen-sm-max-height) {
+    width: auto;
+  }
 
-  .ga-draw-disclaimer a {
-    color: red;
-    text-decoration: none;
+  .ga-draw-info-save {
+    color: lightgray;
+    margin-left: 30px;
 
-    &:hover {
-      text-decoration: underline;
+    @media (max-width: @screen-tablet), (max-height: @screen-sm-max-height) {
+      margin: 0;
+    }
+    @media (max-width: @screen-phone) {
+      height: 20px;
     }
   }
+  .ga-draw-disclaimer {
+    margin-top: 5px;
+    padding-top: 5px; 
+    border-top: 1px solid rgb(238, 238, 238);
+    font-size: 10px;
+
+    a {
+      color: red;
+      text-decoration: none;
 
-  .ga-draw-buttons {
-    position: relative;
-    width: 350px;
-    padding: 0px 0px 5px;
-    
-    button {
-      width: 54px;
-      height: 80px;
-      margin: 0px 2px;
-      padding: 0px;
-      border: none;
-      background: none;
-      
       &:hover {
-        opacity: 0.6;
+        text-decoration: underline;
       }
+    }
+  }
+  
+  .ga-draw-disclaimer, .ga-draw-interaction-share {
+    text-align: justify;
+  }
+
+  .ga-share-permalink {
+    .form-group, .input-group {
+      width: 100%;
+    }
+  }
+   
+  .ga-draw-buttons {
+    margin-bottom: 45px;
 
+    @media (max-width: @screen-tablet), (max-height: @screen-sm-max-height) {
+      margin-bottom: 10px;
+    }
+    
+    > .btn {
+      margin: 5px 30px;
+      width: 75px;
+      height: 75px;
+      padding: 2px;
+      transition: all 0.5s ease;
+      transition-property: width, height, margin;
```

Comment:
Completely off topic. But do you know a good css transition tutorial? They're a mystery to me.